doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2352,6 +2352,15 @@ when using it to configure your core name!
 
 Can also be set via *MicroProfile Config API* sources, e.g. the environment variable ``DATAVERSE_SOLR_PATH``.
 
+dataverse.concurrency.max-async-indexes
```

Comment:
Can we please have a release note snippet where this new config option is mentioned?

Reply:
Note added.

Reply:
@ErykKul thank you! ❤️